rcljava/src/main/java/org/ros2/rcljava/node/Node.java

```diff
@@ -175,15 +177,39 @@
 
   String getName();
 
+  void declareParameter(ParameterVariant parameter);
```

Comment:
rclcpp also returns the value of the declared parameter. If that's possible to do in rcljava, I think it would be convenient when writing application code.

Reply:
I don't feel strongly about it either.  Given the API I've chosen here (taking in a ParameterVariant), the caller already had to do that, so it is somewhat redundant.  But for consistency with the rclcpp and rclpy APIs, I also now return it.  36daa72840b68189f0c6d7d763440c2395ec0d46